submissions/fiyingabriel/htmlForms/signUp.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
```

Comment:
In this file, I don't see where you linked your product page from the html folder, your contact form is not linked on this page and vice versa.

Reply:
@Fiyin985

Reply:
Where did you link the product page in this file? @Fiyin985